gradle.properties

```diff
@@ -1,5 +1,5 @@
 # Project-wide Gradle settings.
-VERSION_NAME=4.5.0-SNAPSHOT
+VERSION_NAME=4.4.1
```

Comment:
This should be `4.5.0` instead of `4.4.1`. We're adding new APIs and make them available. Some APIs weren't included in `4.4.0`. Talked to @osana and we can fix it because `4.4.1` wasn't released yet 😌